Routing

Maccolm opened the pull request.

Maccolm committed:
set routing, created pages
- header
-main page,
-footer

created header

Maccolm committed:
Connecting sidebar

Maccolm committed:
sidebar module

Maccolm committed:
menu bar,
responsive app to 320px
logo
aside menu

Maccolm committed:
styling components

Maccolm committed:
visual part made

Maccolm committed:
stylizing,
pipe-limited symbols,

Maccolm committed:
components... functions

inga-lekhman commented on src/app/pages/movie-favorite-list-page/movie-favorite-list-page.component.ts:
```diff
@@ -0,0 +1,12 @@
+  styleUrl: './movie-favorite-list-page.component.scss'
+})
+export class MovieFavoriteListPageComponent {
+
```
тут вам треба взяти дані з роуту  (id фільмів) і у відповідних мокових даний їх знайти та відобразити в темплейті цієї сторінки

inga-lekhman commented on src/app/pages/movie-header-page/movie-header-page.component.html:
```diff
@@ -0,0 +1 @@
+<app-movie-header [favoriteMoviesIds]="favoritesMoviesListIds" [watchLaterIds]="watchLaterMoviesListIds"></app-movie-header>
```
ця сторінка не потрібна

inga-lekhman reviewed (commented):
По-перше, не потрібно назви всіх компонентів починати з слова movie))це пекло для очей і швидкого пошуку файлів. 

Ви трохи наплутали з структурою. І кожен компонент зробили як сторінку.
Сторінок у вас має бути 6, як пунктів меню(в сайдбар 4 і в хедері 2)
- NowPlayingPage;
- TopRatePage;
- PopularPage;
- UpcomingPage;
- FavoritePage;
- WatchListPage;


Всі інші сторінки видаліть)

Компонент header, має бути так як ви його використали на сторінці popular по аналогії на сторінках NowPlayingPage, TopRatePage, UpcomingPage;

Сподіваюсь це вам трохи допоможе розібратись

inga-lekhman commented on src/app/app.component.html:
```diff
@@ -1,2 +1,21 @@
+</main>
+<footer>
+	footer
+</footer>
```
подивіться мій репозиторій, там ви побачите структуру цієї сторінки https://github.com/inga-lekhman/angular-course/blob/main/src/app/app.component.html 
А також компонентів і сторінок. Та ще разок відео собі включіть, початок минулого уроку де ми розбирали це дз)

Maccolm committed:
fixed routes,
- added logic to receive and transfer information through router link
- made details window
- favorite and watch list page work and receiving id from other pages.
